the-blue-alliance-ios/EventsViewController.m

```diff
@@ -37,10 +38,8 @@ - (void)setWeeks:(NSArray<NSNumber *> *)weeks {
     _weeks = weeks;
     
     if (weeks && !self.currentWeek) {
-        NSNumber *week = weeks.firstObject;
-        
-        self.currentWeek = week;
-        self.eventsViewController.week = week;
+        self.currentWeek = @([EventWeek eventOrderForDate:[NSDate date]]);
```

Comment:
If this year is not the current year, can we default to the first week of competition?